mailpoet/lib/Newsletter/Scheduler/PostNotificationScheduler.php

```diff
@@ -95,7 +108,11 @@ public function schedulePostNotification($postId) {
     }
   }
 
-  public function createPostNotificationSendingTask(NewsletterEntity $newsletter): ?SendingTask {
+  /**
```

Comment:
I guess this annotation is redundant. In my opinion, the return type and parameter type are enough.